src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -28,10 +28,12 @@ public LineService(LineRepository lineRepository, StationService stationService)
 
     public LineResponse saveLine(LineRequest request) {
         Line line = lineRepository.save(new Line(request.getName(), request.getColor()));
-        if (request.getUpStationId() != null && request.getDownStationId() != null && request.getDistance() != 0) {
+        if (request.getUpStationId() != null && request.getDownStationId() != null
+                && request.getDistance() != 0) {
             Station upStation = stationService.findById(request.getUpStationId());
             Station downStation = stationService.findById(request.getDownStationId());
-            line.getSections().add(new Section(line, upStation, downStation, request.getDistance()));
+
+            line.addSection(new Section(line, upStation, downStation, request.getDistance()));
```

Comment:
```suggestion
            line.addSection(upStation, downStation, request.getDistance());
```
line 객체 자기 자신을 다시 파라미터로 전달할 필요 없이 section 객체 생성에 필요한 값만 전달해보면 어떨까요